src/model_selection/mod.rs

```diff
@@ -130,11 +130,13 @@ pub trait BaseKFold {
 /// * `y` - target values, should be of size _N_
 /// * `test_size`, (0, 1] - the proportion of the dataset to include in the test split.
 /// * `shuffle`, - whether or not to shuffle the data before splitting
+/// * `seed` - Controls the shuffling applied to the data before applying the split. Pass an int for reproducible output across multiple function calls
 pub fn train_test_split<T: RealNumber, M: Matrix<T>>(
     x: &M,
     y: &M::RowVector,
     test_size: f32,
     shuffle: bool,
+    seed: Option<u64>,
```

Comment:
Backward incompatible change that will require bump of version to `0.3`

Reply:
To avoid breaking the API, an alternative would be to have a static rng, that can be seeded with an api call, and we could make that call at the start of tests.

Reply:
I think that we are going to do breaking changes in #108, so in my opinion we should not be worried about adding backward incompatible changes like this one as long as we document it in a changelog and we bump the version appropriately